css/index.css

```diff
@@ -51,30 +51,47 @@ table {
     box-sizing: border-box;
 }
 
-.toptop {
-    width: 80%;
-    height: 50px;
+.toptop nav {
     margin: 30px;
-    color: black;
     background: white;
-    text-decoration: none;
-    font-size: 1.3rem;
+
+    display: flex;
+
+
 }
 
```

Comment:
There is some minor CSS tweaks you can make to get your header/nav spaced the same as the rest of the content on the page.